owsi-core/owsi-core-components/owsi-core-component-jpa/src/main/java/fr/openwide/core/jpa/batch/executor/BatchRunnable.java

```diff
@@ -0,0 +1,32 @@
+package fr.openwide.core.jpa.batch.executor;
+
+import java.util.List;
+
+public abstract class BatchRunnable<E> {
```

Comment:
If we use an interface (on the AbstractBatch/AbstractBatchExecutor) and provide an empty implementation (say AbstractBatchRunnable), then when we'll use Java 8, it will be lambda-ready.
Are you ok with this?